File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ৬ — যোগফল নির্ণয়.java
import java.util.Scanner;

class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int T, N, x, y, total;
        
        T = sc.nextInt();  // Read number of test cases
        
        for (int i = 1; i <= T; i++) {  
            N = sc.nextInt();  // Read value of N for each test case
            
            x = N % 10;         // Extract last digit
            y = N / 10000;      // Extract first digit (for a 5-digit number)
            total = x + y;      // Calculate sum
            
            System.out.println("Sum = " + total); // Print result
        }
        
        sc.close(); // Close Scanner to prevent resource leaks
    }
}

File: ৫২টি প্রোগ্রামিং সমস্যা ও সমাধান/প্রোগ্রামিং সমস্যা ৬ — যোগফল নির্ণয়.py
T = int(input())  # Take number of test cases

for i in range(1, T + 1):  # Loop through test cases
    N = int(input())  # Input value for N
    
    x = N % 10  # Get last digit
    y = N // 10000  # Get first digit (for a 5-digit number)
    
    total = x + y  # Avoid using 'sum' as a variable name (it's a built-in function)
    print("Sum =", total)  # Removed 'end=' ' to ensure proper line breaks
